search/mindbender-search

```diff
@@ -163,9 +170,9 @@ case $Action in
             # show progress
             pv --line-mode --size="$(deepdive sql eval "SELECT COUNT(*) FROM $relation")" |
             # split records into multiple batches
-            # TODO parallelize
-            split --lines=$ELASTICSEARCH_BULK_BATCHSIZE --filter="$(escape-args-for-shell sh -euc '
-                jqToFormat=$1 indexURL=$2
+            # TODO parallelize            
+       split --lines=$ELASTICSEARCH_BULK_BATCHSIZE --suffix-length=5 --filter="$(escape-args-for-shell sh -euc '
```

Comment:
5 seems to be better than the default 2, but this isn't a true fix.  In fact, we can always determine a safe `--suffix-length` in this case as the number of input lines and the `$ELASTICSEARCH_BULK_BATCHSIZE` are known.  `--numeric-suffixes` may help as you can take the length of the number of splits in decimals without bothering with the number of alphabets.